config/checkstyle/checkstyle.xml

```diff
@@ -34,7 +34,7 @@
   </module>
   <module name="SuppressWarningsFilter" />
   <module name="LineLength">
-    <property name="max" value="120"/>
+    <property name="max" value="180"/>
```

Comment:
Changing this checkstyle here was my initiative, but I want it to be clear that we shouldn't.

Also: I'm doing it as a TL. As devs, DO NOT modify this file of your own initiative, please.